src/core/thread/package.d

```diff
@@ -0,0 +1,16 @@
+/**
+ * The thread module provides support for thread creation and management.
+ *
+ * Copyright: Copyright Sean Kelly 2005 - 2012.
+ * License: Distributed under the
+ *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Authors:   Sean Kelly, Walter Bright, Alex Rønne Petersen, Martin Nowak
+ * Source:    $(DRUNTIMESRC core/thread/package.d)
+ */
+
+module core.thread;
+
+public import core.time;  // Should be removed
```

Comment:
can these be `package(core.thread) import`s?

Reply:
Let me post the PM:

The initial `thread.d` had `public import core.time`. So, I removed `public` but then `spinlock` (iirc) used something of `core.time` which was coming from `thread`.
It was kind of a mess so I removed the `public` from `core.thread` and imported on `spinlock`.
As you can see, I get an `undefined reference` error which I don't understand. Thinking that it might have been a problem of with removing the `public import` from `core.thread`, I put it again on `package.d`.

Tl;dr, the `public import core.time` should not be needed. The rest yes, they can.